ci/dram.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Need to add a job to the .gitlab_ci.yaml to run this (can add to misc stage)

Reply:
updated the gitlab file. can you take a look?